website_media_size/static/src/js/website_media_size.js

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2018 Onestein
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+odoo.define('website_media_size', function(require) {
+    "use strict";
+
+    var web_editor_widget = require('web_editor.widget');
+    var MediaDialog = web_editor_widget.MediaDialog;
+    var ImageDialog = web_editor_widget.ImageDialog;
+
+    MediaDialog.include({
+        xmlDependencies: MediaDialog.prototype.xmlDependencies.concat(
+            ['/website_media_size/static/src/xml/website_media_size.xml']
+        ),
+        start: function() {
+            var res = this._super.apply(this, arguments);
+            this.search();
```

Comment:
Hmm why this?

Also why patching different methods on different classes? I'm not saying it's wrong, just weird at 1st sight... :thinking:

Reply:
You're right this is not needed anymore. I removed it and moved the xml dependency to ImageDialog.